### ISSUE TITLE
December release notes

### ISSUE THREAD

**Issue by olegmoz** [artipie/blog.artipie.com#16]
We need to publish December release notes article to give overview of changes made in Artipie during the month

**Comment by olegmoz** [comment]
@olenagerasimova here is list of pull requests for last 30 days:
```
 - #150 - Fix bug in meta skeleton @genryxy: https://github.com/artipie/npm-adapter/pull/151
 - #119 - Fix backward compatibility @genryxy: https://github.com/artipie/npm-adapter/pull/149
 - #296 - Copy all keys @olegmoz: https://github.com/artipie/asto/pull/297
 - #829 - docker-adapter upgraded to 0.10 @olegmoz: https://github.com/artipie/artipie/pull/830
 - #826 - Verbose output for IT cases @genryxy: https://github.com/artipie/artipie/pull/827
 - #797 - Update management-api version, use ConfigFileApi @genryxy: https://github.com/artipie/artipie/pull/824
 - #822 - Fix HelmIT @genryxy: https://github.com/artipie/artipie/pull/823
 - #820 - docker-adapter upgraded to 0.9.3 @olegmoz: https://github.com/artipie/artipie/pull/821
 - #804 - update readme @olenagerasimova: https://github.com/artipie/artipie/pull/817
 - #797 - Support both extensions for ArtipieApi and DashboardSlice @genryxy: https://github.com/artipie/artipie/pull/816
 - #804 - Add setting for repo configs location @olenagerasimova: https://github.com/artipie/artipie/pull/815
 - #797 - Use ConfigFile where RxStorage is used for config files @genryxy: https://github.com/artipie/artipie/pull/814
 - #804 - handle exception on repo config parsing @olenagerasimova: https://github.com/artipie/artipie/pull/811
 - #797 - Get value for different extensions  @genryxy: https://github.com/artipie/artipie/pull/810
 - #803 - specify layout on start @olenagerasimova: https://github.com/artipie/artipie/pull/809
 - artipie/management-api#15 - update management API and use GetStorageSlice from there @olenagerasimova: https://github.com/artipie/artipie/pull/808
 - #797 - Check existing, trim filename for different extensions @genryxy: https://github.com/artipie/artipie/pull/807
 - #738 - Fix path handling for NpmProxySlice @genryxy: https://github.com/artipie/artipie/pull/805
 - #775 - test for ArtipieApi @olenagerasimova: https://github.com/artipie/artipie/pull/801
 - #798 - update management-api @olenagerasimova: https://github.com/artipie/artipie/pull/799
 - #793 - prepare to transfer dashboard @olenagerasimova: https://github.com/artipie/artipie/pull/796
 - #778 - Add tests for YamlProxyConfig, extract class StorageYamlConfig @genryxy: https://github.com/artipie/artipie/pull/794
 - #597 - Add negative tests to RepositoriesFromStorage @genryxy: https://github.com/artipie/artipie/pull/792
 - #586 - Layout extracted @olegmoz: https://github.com/artipie/artipie/pull/785
 - #597 - Test RepositoriesFromStorage @genryxy: https://github.com/artipie/artipie/pull/783
 - #768 - Support Basic auth for Gem @genryxy: https://github.com/artipie/artipie/pull/782
 - Simplify YamlSettings.java @portlek: https://github.com/artipie/artipie/pull/780
 - #778 - Extend tests for RepoConfig @genryxy: https://github.com/artipie/artipie/pull/779
 - #422 - Handling empty UUID @olegmoz: https://github.com/artipie/docker-adapter/pull/425
 - #372 - Tags implemented in MultiReadManifests @olegmoz: https://github.com/artipie/docker-adapter/pull/423
 - #366 - Tags implemented in CacheManifests @olegmoz: https://github.com/artipie/docker-adapter/pull/421
 - #417 - Parsing scope from string @olegmoz: https://github.com/artipie/docker-adapter/pull/420
 - #366 - Added JoinedTagsSource @olegmoz: https://github.com/artipie/docker-adapter/pull/419
 - #417 - Added authentication Scope @olegmoz: https://github.com/artipie/docker-adapter/pull/418
 - #369 - Tags implemented in ReadWriteManifests @olegmoz: https://github.com/artipie/docker-adapter/pull/416
 - #366 - Added TagsPage @olegmoz: https://github.com/artipie/docker-adapter/pull/415
 - #366 - Added ParsedTags @olegmoz: https://github.com/artipie/docker-adapter/pull/414
 - #393 - Mount implemented in ReadWriteLayers @olegmoz: https://github.com/artipie/docker-adapter/pull/413
 - #375 - Tags implemented in ProxyManifests @olegmoz: https://github.com/artipie/docker-adapter/pull/412
 - #375 - Added TagsListUri @olegmoz: https://github.com/artipie/docker-adapter/pull/411
 - #368 - Catalog in CacheDocker @olegmoz: https://github.com/artipie/docker-adapter/pull/410
 - #368 - Catalog in TrimmedDocker @olegmoz: https://github.com/artipie/docker-adapter/pull/409
 - #367 - JoinedCatalogSource extracted from MultiReadDocker @olegmoz: https://github.com/artipie/docker-adapter/pull/408
 - #371 - Catalog implemented in MultiReadDocker @olegmoz: https://github.com/artipie/docker-adapter/pull/407
 - #405 - Support for manifest conversion to wildcard @olegmoz: https://github.com/artipie/docker-adapter/pull/406
 - #371 - CatalogPage extracted from AstoCatalog @olegmoz: https://github.com/artipie/docker-adapter/pull/404
 - #371 - ParsedCatalog added @olegmoz: https://github.com/artipie/docker-adapter/pull/403
 - #370 - Catalog implemented in ReadWriteDocker @olegmoz: https://github.com/artipie/docker-adapter/pull/402
 - #376 - Catalog proxy @olegmoz: https://github.com/artipie/docker-adapter/pull/401
 - #376 - Extracted ResponseSink from ProxyLayers and ProxyManifests @olegmoz: https://github.com/artipie/docker-adapter/pull/400
 - #376 - Added CatalogUri @olegmoz: https://github.com/artipie/docker-adapter/pull/399
 - #297 - Invalid digest error on blob creation @olegmoz: https://github.com/artipie/docker-adapter/pull/398
 - #297 - InvalidDigestException added @olegmoz: https://github.com/artipie/docker-adapter/pull/397
 - #390 - Blob mount endpoint implementation @olegmoz: https://github.com/artipie/docker-adapter/pull/396
 - #394 - Mount implemented in AstoLayers @olegmoz: https://github.com/artipie/docker-adapter/pull/395
 - #390 - Added Layers.mount method without implementation @olegmoz: https://github.com/artipie/docker-adapter/pull/392
 - #390 - Mount blob request parameters parsing @olegmoz: https://github.com/artipie/docker-adapter/pull/391
 - #374 - Pagination support in AstoCatalog @olegmoz: https://github.com/artipie/docker-adapter/pull/389
 - #275 - Content constructors in RsWithBody and RsFull @olegmoz: https://github.com/artipie/http/pull/282
 - #278 - AuthScheme.Fake added @olenagerasimova: https://github.com/artipie/http/pull/279
 - #276 - RqParams class added @olegmoz: https://github.com/artipie/http/pull/277
 - #273 - JoinedPermissions added @olegmoz: https://github.com/artipie/http/pull/274
 - #83 - Remove deprecated auth classes @genryxy: https://github.com/artipie/gem-adapter/pull/84
 - #81 - Support Basic authentication @genryxy: https://github.com/artipie/gem-adapter/pull/82
 - #81 - Fix ITCase @genryxy: https://github.com/artipie/helm-adapter/pull/82
 - #57 - Add IT with authentication @genryxy: https://github.com/artipie/helm-adapter/pull/80
 - #14 - November blog post @olenagerasimova: https://github.com/artipie/blog.artipie.com/pull/15
 - #27 - Revert using ContentAs in ApiRepoGetSlice @genryxy: https://github.com/artipie/management-api/pull/33
 - #27 - Use ConfigFiles in page implementations @genryxy: https://github.com/artipie/management-api/pull/32
 - #27 - Use ConfigFile in RepoUpdateSlice, remove ApiRepoCreateSlice @genryxy: https://github.com/artipie/management-api/pull/30
 - #27 - ConfigFile for RepoGet and RepoList slices @genryxy: https://github.com/artipie/management-api/pull/29
 - #27 - ConfigFile for permissions slices @genryxy: https://github.com/artipie/management-api/pull/28
 - #15 - GetStorageSlice migrated @olenagerasimova: https://github.com/artipie/management-api/pull/26
 - #24 - artifactory API endpoints to ApiPermission @olenagerasimova: https://github.com/artipie/management-api/pull/25
 - #16 - dashboard classes should be public @olenagerasimova: https://github.com/artipie/management-api/pull/23
 - #14 - extracted interface from Cookies @olenagerasimova: https://github.com/artipie/management-api/pull/19
 - #16 - transfer dashboard @olenagerasimova: https://github.com/artipie/management-api/pull/18
 - #11 - support xz compression @olenagerasimova: https://github.com/artipie/debian-adapter/pull/16
 - #11 - read control file @olenagerasimova: https://github.com/artipie/debian-adapter/pull/15
 - #13 - expand readme @olenagerasimova: https://github.com/artipie/debian-adapter/pull/14
 - #2 - added test for install @olenagerasimova: https://github.com/artipie/debian-adapter/pull/10
 - #2 - test for search @olenagerasimova: https://github.com/artipie/debian-adapter/pull/9
 - #6 - github actions @olenagerasimova: https://github.com/artipie/debian-adapter/pull/7
 - #1 - project setup @olenagerasimova: https://github.com/artipie/debian-adapter/pull/5
```

**Comment by olenagerasimova** [comment]
closed by #17
